torchgen/gen_functionalization_type.py

```diff
@@ -776,7 +781,10 @@ def gen_functionalization_definition(
         # I think we should either:
         # (1) fix their schemas (BC-breaking)
         # (2) hand-write their functionalization kernels
-        if str(g.func.name) not in MUTABLE_OPS_NOT_USING_FUNCTIONALIZATION:
+        if (
+            str(g.func.name) not in MUTABLE_OPS_NOT_USING_FUNCTIONALIZATION and
+            str(g.func.name.name) not in MUTABLE_OPS_NOT_USING_FUNCTIONALIZATION
```

Comment:
Maybe better to just special case set_ here directly rather than adding it to `MUTABLE_OPS_NOT_USING_FUNCTIONALIZATION`?

Reply:
tbh I think this is fine - we do the exact same thing with `resize_()` (where instead of having it go through all of this codegen, we just give it a dedicated, hand-written functionalization kernel here: https://github.com/pytorch/pytorch/blob/main/aten/src/ATen/FunctionalizeFallbackKernel.cpp#L329)